src/GeositeFramework/Views/Home/Index.cshtml

```diff
@@ -286,10 +288,10 @@
     </script>
 
     <script type="text/template" id="template-sidebar-plugin">
-            <a class="plugin-launcher i18n" data-i18n="[title]<%=fullName%>" href="javascript:;">
-                <img src="<%- pluginSrcFolder %>/icon<%= selected ? '_active' : ''%>.png">
-                <div class="i18n" data-i18n="<%- pluginObject.toolbarName %>"><%- pluginObject.toolbarName %></div>
-            </a>
+        <button class="nav-apps-button" i18n" data-i18n="[title]<%=fullName%>">
```

Comment:
I suspect this is the same in the current branch, but when these plugin items are rendered they appear to get wrapped in an `<li>` tag which isn't present in the wireframes.  It looks like they have some attributes on them that is important, but they don't seem to get written inside of a `<ul>` tag.  If it's not too costly, consider adding the appropriate container element.

![screenshot 2016-10-17 at 1 26 17 pm](https://cloud.githubusercontent.com/assets/1014341/19447879/629f0028-946d-11e6-8403-27700dbedb58.png)